apps/aechannel/src/aesc_fsm.erl

```diff
@@ -425,7 +431,7 @@ awaiting_leave_ack(cast, {?LEAVE_ACK, Msg}, D) ->
             close(Err, D)
     end;
 awaiting_leave_ack(timeout, awaiting_leave_ack = T, D) ->
```

Comment:
This function clause could be removed entirely because it is handled further down by `handle_common_event` too.